niyoceles/banka
# [Chore #165177435] API Endpoints environment configuration

#### What does this PR do?
API endpoints environment configuration
#### Description of Task to be completed?
API endpoints configuration by install the tools required
#### How should this be manually tested?
After cloning  a repo, should run the following command:
npm install
npm run dev
then use postman to test:
localhost:3000/api/v1/home
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
#165177435
